### ISSUE TITLE
Search function not working (and images)

### ISSUE THREAD

**Issue by ebrondol** [matcornic/hugo-theme-learn#149]
Dear developers,

I am completely new in websites and this is my first with hugo. I liked very much your theme and I am creating my website using it.
My developments can be found in GitLab : https://gitlab.cern.ch/ericabro/HGCAL_sensor_web/tree/HugoDev/hugo_folder
And I am working with the following specs:
HUGO VERSION = Hugo Static Site Generator v0.40.1 linux/amd64 BuildDate: 2018-04-25T17:16:11Z
CHROME VERSION = Google Chrome 67.0.3396.87 

There are two problems that I have and it would be very useful for me to get some help.
- Unfortunately I did not menage to set the Search function active yet. I modified the `config.toml` as suggested but as you can see from the following picture this does not work:
![screenshot from 2018-07-18 12-04-28](https://user-images.githubusercontent.com/5331004/42875113-276d35de-8a83-11e8-8aa3-b12d48ab71d6.png)
I have also tried to play a bit with `lunr-hugo` and produce the JSON file in `static/json/search.json` but this seems also a dead road. Any suggestion?
- I would like to put two pictures on the same line. I have tried with some suggestions I found on 
http://yoshiharuyamashita.com/post/hugo-shortcode-to-show-multiple-images/
using a new shortcode `fluid_imgs` but it does not work. Did you ever try to do this in your theme? Maybe do you have some suggestions?

Thank you very much in advance.

**Comment by kamar535** [comment]
I'm using Google Chrome Version 68.0.3440.106 (Official Build) (64-bit).

For this [site](http://www.it.uu.se/education/course/homepage/os/vt18/) I built in the spring of 2018 site search works as expected. 

Now I'm using Hugo v0.41 and the following version of Hugo Learn Theme:

```
commit 59c4cbf18690feb0cd0a16ad0f1434249c9dd5f3 (HEAD -> master, origin/master, origin/HEAD)
Author: matcornic <mathieu.cornic@gmail.com>
Date:   Tue Feb 20 19:47:55 2018 +0100

    fix #130: favicon works now with baseURL
```
, and cannot get site search to work on a [new site](http://www.it.uu.se/education/course/homepage/introdat/ht18/) I'm developing.

I've also noted that the search on the official [home page for the Hugo Learn Theme](https://themes.gohugo.io/theme/hugo-theme-learn/en/) no longer works. 

For both sites I have the activated search as described [here](https://themes.gohugo.io/theme/hugo-theme-learn/en/basics/configuration/#activate-search) by adding the following to the `config.toml` file. 


```
[outputs]
home = [ "HTML", "RSS", "JSON"]
```

**Comment by matalo33** [comment]
Thanks for pointing out search not working on the themes website, created #173 to track this.

Closing this ticket due to old age. If you're still having issues please write back.
